src/main/java/foundationgames/enhancedblockentities/EBESetup.java

```diff
@@ -27,7 +27,7 @@
 import net.minecraft.util.DyeColor;
 import net.minecraft.util.Identifier;
 
-public enum EBESetup {;
+public class EBESetup {
```

Comment:
Out of curiosity, why make this change? (the enum thing is a dirty way I made this an uninstantiable and unextendable utility class)

Reply:
It seemed like a mistake lol, reverted